src/icon-manifest/supportedExtensions.ts

```diff
@@ -178,7 +178,8 @@ export const extensions: IFileCollection = {
       format: FileFormat.svg,
       disabled: true,
     },
-    { icon: 'docpad', extensions: ['eco'], light: true, format: FileFormat.svg },
```

Comment:
why did you removed this line?

Reply:
Oh shit. Nice catch. Ment to do a copy-paste but ended up deleting the line.